### PR TITLE
Synchronization with BUT internal git

### PR DESCRIPTION
**Pull request by Kapim**

### CODE REVIEW

**Comment by Kapim** [comment]
@ZdenekM could you please take a quick look to see if I've messed anything up? Thank you

**Review by ZdenekM — APPROVED** [review]
No obvious problem was found.

File: src/python/era_5g_netapp_client/era_5g_netapp_client/client.py [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+                print(self.resource_checker.is_ready())
+                self.load_netapp_uri()
+            print('Feedback: Got new plan successfully.')
+        except Exception as ex:
+            self.deleteAllResources()
+            raise ex
+
+    
+        
+    def register(self, args=dict()) -> str:
```
Comment (ZdenekM) [inline src/python/era_5g_netapp_client/era_5g_netapp_client/client.py:115]: @Kapim Check this https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python

File: src/python/era_5g_object_detection_distributed_interface/era_5g_object_detection_distributed_interface/interface.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+@app.route('/register', methods=['GET'])
+def register():
+    """
+    Needs to be called before an attempt to open WS is made.
+
+    Returns:
+        _type_: The port used for gstreamer communication.
+    """
+    global logger
```
Comment (ZdenekM) [inline src/python/era_5g_object_detection_distributed_interface/era_5g_object_detection_distributed_interface/interface.py:44]: @Kapim `global` should not be needed here (only if you want to assign something).

Comment (Kapim) [reply src/python/era_5g_object_detection_distributed_interface/era_5g_object_detection_distributed_interface/interface.py:44]: removed in multiple locations, thanks

File: src/python/era_5g_netapp_client/era_5g_netapp_client/client.py (hunk 1 above)
Comment (Kapim) [reply src/python/era_5g_netapp_client/era_5g_netapp_client/client.py:115]: Wasn't aware of this, thank you, changed to None instead.

**Pull request merged**